netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -58,6 +60,12 @@
     private final transient Base base;
 
     /**
+     * Token cache.
+     */
+    private final ConcurrentHashMap<String, Object[]> tokencache = new
```

Comment:
@prahladyeri a couple of comments here:

1. Variable should be declared as `ConcurrentMap` interface type instead of using concrete class `ConcurrentHashMap`.
2. [Avoid compound names](http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html), we can call this `tokens` instead.